secant/Util/DatabaseFiles.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  DatabaseFiles.swift
+//  secant-testnet
+//
+//  Created by Lukáš Korba on 05.04.2022.
+//
+
+import Foundation
+
+struct DatabaseFiles {
+    enum DatabaseFilesError: Error {
+        case documentsURL
```

Comment:
The error types are not indicative of what's going on. Could you give them a more meaningful name? Mostly the first two. Could mean anything.